app/src/test/java/org/simple/clinic/bp/history/adapter/BloodPressureHistoryListItemTest.kt

```diff
@@ -0,0 +1,81 @@
+package org.simple.clinic.bp.history.adapter
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.simple.clinic.patient.PatientMocker
+import org.simple.clinic.util.TestUtcClock
+import org.threeten.bp.Duration
+import org.threeten.bp.Instant
+import org.threeten.bp.temporal.ChronoUnit.MINUTES
+import java.util.UUID
+
+class BloodPressureHistoryListItemTest {
+
+  @Test
+  fun `if blood pressure is editable, then user can edit blood pressure`() {
+    // when
+    val createdAt = Instant.parse("2020-01-01T00:00:00Z")
+    val bloodPressureNow = PatientMocker.bp(
+        uuid = UUID.fromString("78e876ae-3055-43d6-a132-7ad5dd930e23"),
+        systolic = 120,
+        diastolic = 70,
+        createdAt = createdAt
+    )
+    val bloodPressureADayInFuture = PatientMocker.bp(
+        uuid = UUID.fromString("2b929a33-e543-4a4f-a98e-5d0d3e8e1e03"),
+        systolic = 120,
+        diastolic = 70,
+        createdAt = createdAt.minus(15, MINUTES)
+    )
+    val utcClock = TestUtcClock(Instant.parse("2020-01-01T00:00:00Z"))
+    val listItems = BloodPressureHistoryListItem.from(listOf(bloodPressureNow, bloodPressureADayInFuture), Duration.ofMinutes(10), utcClock)
+
+    // then
+    assertThat(listItems)
+        .containsExactly(
+            BloodPressureHistoryListItem(measurement = bloodPressureNow, isBpEditable = true, isHighBloodPressure = false, showDivider = true),
+            BloodPressureHistoryListItem(measurement = bloodPressureADayInFuture, isBpEditable = false, isHighBloodPressure = false, showDivider = false)
+        )
+  }
+
+  @Test
+  fun `if blood pressure is high, then show high blood pressure`() {
```

Comment:
Reword this test?

Ex: `if the blood pressure reading is high, then it must be displayed as a high BP`?